lib/provisioning/ProvisionDispatcher.js

```diff
@@ -76,6 +76,22 @@ class ProvisionDispatcher {
         ], cb);
     }
 
+    checkState(cb, autoTrigger) {
+        this._onLeaderChange(autoTrigger);
+        this._onOwnerChange(autoTrigger);
+        this._onMyselfChange(cb, autoTrigger);
+        this._onProvisionChange(autoTrigger);
+        return undefined;
+    }
+
+    periodicCheckState(cb) {
+        setTimeout(() => {
```

Comment:
Instead of re-calling `periodicCheckState` at the end, you could use `setInterval` in place of `setTimeout`